examples/ibm-cis/main.tf

```diff
@@ -460,4 +460,23 @@ resource "ibm_cis_logpush_job" "test" {
 data "ibm_cis_logpush_jobs" "test" {
     cis_id          = data.ibm_cis.cis.id
     domain_id       = data.ibm_cis_domain.cis_domain.domain_id
-}
\ No newline at end of file
+}
+
+#CIS MTLS instance
+resource "ibm_cis_mtls" “test” {
+  cis_id          = ibm_cis.web_domain.id
+  domain_id       = ibm_cis_domain.web_domain.id
+  cert_mtls       = "-----BEGIN CERTIFICATE----- -----END CERTIFICATE-----"
```

Comment:
Instead of certifcate in one line you can use the multiline in
```
  cert_mtls = <<EOT
----------(certificate)
EOT
```

Reply:
OK.